tools/tf2openapi/cmd/main.go

```diff
@@ -31,7 +32,7 @@ func main() {
 	model := UnmarshalSavedModelPb(modelPb)
 
 	/** Schema generation example **/
-	spec, err := generator.GenerateOpenAPI(model)
+	spec, err := generator.GenerateOpenAPI(model, *sigDefKey)
```

Comment:
Does this *have* to be specified. Will this break if specified?

Reply:
No, it doesn't have to be specified. If the key isn't specified, the generator defaults to "serving_default"